protocols/kad/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.45.0 - unreleased
+
+- Emit `ModeChanged` event whenever we automatically reconfigure the mode.
```

Comment:
```suggestion
- Emit `ModeChanged` event whenever we automatically reconfigure the mode.
  See [PR 4341].
```

References alone don't do anything in markdown, we need to mention them somewhere :)